static/js/apps/nl_interface/data_app/app.tsx

```diff
@@ -155,6 +158,10 @@ export function App(): JSX.Element {
               inputNextPrompt();
             }
           }}
+          indexType={indexType}
+          useLLM={useLLM}
+          setIndexType={(v) => setIndexType(v)}
```

Comment:
nit: can actually just do `setIndexType={setIndexType}` because setIndexType is technically of the type (v: string) => void

& for same for setUseLLM

Reply:
Done.

Reply:
this doesn't look updated?

Reply:
Oh sorry, I updated the other one and missed this.